pegasus/sites.v3/code.org/public/music/index.haml

```diff
@@ -15,6 +15,8 @@ theme: responsive_full_width
         =hoc_s("music_lab.jam_session.banner.desc_01")
       %a.link-button.white{href: CDO.studio_url("/s/music-jam-2024/reset")}
         =hoc_s("music_lab.jam_session.button.start")
+      %a.link-button.secondary.white{href: CDO.studio_url("/projects/music/new")}
+        =hoc_s("music_lab.jam_session.button.create_new_project")
```

Comment:
You can add a `.button-wrapper` and gap styles to increase the column spacing to align with our grid standards:
```suggestion
     .button-wrapper{style: "column-gap: 1rem; row-gap: 0"}
        %a.link-button.white{href: CDO.studio_url("/s/music-jam-2024/reset")}
          =hoc_s("music_lab.jam_session.button.start")
        %a.link-button.secondary.white{href: CDO.studio_url("/projects/music/new")}
          =hoc_s("music_lab.jam_session.button.create_new_project")
```
<img width="1376" alt="Screenshot 2025-01-10 at 8 57 56 AM" src="https://github.com/user-attachments/assets/d44b527e-716a-4dbd-86f3-b77ea479f4cb" />

Reply:
Thanks.  Updated!